config/initializers/clowder_config.rb

```diff
@@ -44,15 +49,20 @@ def self.instance
         options["databasePort"] = ENV['DATABASE_PORT']
         options["databaseUsername"] = ENV['DATABASE_USER']
         options["databasePassword"] = ENV['DATABASE_PASSWORD']
+        options["endpoints"] = {
+          "ingress-services" => "https://ci.cloud.redhat.com",
+          "souces-api"       => ENV["SOURCES_URL"]
+        }
       end
 
       options["APP_NAME"] = "catalog-inventory"
       options["PATH_PREFIX"] = "api"
 
-      options["SOURCES_URL"] = ENV["SOURCES_URL"]
+      options["SOURCES_URL"] = options["endpoints"]["sources-api"]
+      options["UPLOAD_URL"] = "#{options["endpoints"]["ingress-service"]}/api/ingress/v1/upload"
```

Comment:
If `ingress` service fails to deploy, `options["endpoints"]["ingress-service"]` will be empty. We should handle this case

Reply:
Just log the missing endpoint?

Reply:
Yes, it will clear if thing goes wrong

Reply:
logged error